prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -3276,6 +3389,94 @@ subroutine initProgramVariables(input, env)
 
     end if
 
+    if (allocated(reks)) then
+
```

Comment:
Please factor out the entire block into a subroutine (e.g. `printReksInfo()`). I know, that all other components are also not factored out into routines, but we badly need a modularization of the initialization, so let's not make the situation worse by dumping further tons of code in it. :wink:

Reply:
Done with commit a8cb284 to factor out this part, please see this commit carefully.